docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+R is a free, open source, programming language and software environment for statistical computing and graphics developed at Bell Laboratories as a GNU project. It is designed as a true computer language, and is written in C, Fortran and R.
+
+R provides a wide variety of statistical (linear and nonlinear modelling, classical statistical tests, time-series analysis, classification, clustering, etc) and graphical techniques, and is highly extensible.
+
+One of its strengths is the ease with which well-designed publication-quality plots can be produced, including mathematical symbols and formulae where needed. Great care has been taken over the defaults for the minor design choices in graphics, but the user retains full control. Along with this, for computationally-intensive tasks, C, C++ and Fortran can be linked and called at runtime or one can even write C code to manipulate R objects directly.
+
+The R software suite includes:
+An effective data handling and storage facility,
+A suite of operators for calculations on arrays, in particular matrices,
```

Comment:
If you want to make this a list, you'll need to put either `-` or `*` in front of each list item.